English translation (#71)
personium/new-docs: src/getting-started/setup-sample-apps.md

---
id: setup-sample-apps

> title: Building the sample app

Reviewer: ```suggestion
title: Building a sample app
```

sidebar_label: Building a sample app
---

The Personium community publishes a sample application using Personium as open source. These constructions are not essential, but by making them work, you can get an image of an application using Personium and use it as a reference for development.

Please refer to the document on the app repository for the procedure to build each app.

## Unit Manager

This is a sample application that manages data on Personium.

* [Unit Manager](https://github.com/personium/app-uc-unit-manager)

> ## Home app

Reviewer: ```suggestion
## HomeApp
```

> This is a sample application for centrally managing an application that uses Personium and performing Single Sign On.

Reviewer: ```suggestion
This is a sample application for managing applications that uses Personium and launch such applications using Single Sign On flow.
```

> * [Home App](https://github.com/personium/app-cc-home)

Reviewer: ```suggestion
* [HomeApp](https://github.com/personium/app-cc-home)
```

> All apps introduced below can be authenticated and activated from the home app.

Reviewer: ```suggestion
All apps introduced below can be authenticated and launched from the home app.
```

> ## My Board

Reviewer: ```suggestion
## MyBoard
```

This is a sample app that shares text with other people.

> * [My Board](https://github.com/personium/app-myboard)

Reviewer: ```suggestion
* [MyBoard](https://github.com/personium/app-myboard)
```

## Personium Calendar

This is a sample application that aggregates and manages data on existing calendar services such as Google Calendar and Office 365 on PDS.

* [Personium Calendar](https://github.com/personium/app-personium-calendar)

## Personium Trails

> This is a sample application that captures the movement history data accumulated on Google into PDS and discloses a part of the data to others.

Reviewer: ```suggestion
This is a sample application that captures the location history data accumulated on Google into PDS and allow user to discloses selected data to third parties.
```

* [Personium Trails](https://github.com/personium/app-personium-trails)

## Timefiller

> This is a sample application that recommends event information based on the schedule information and profile information accumulated on PDS in conjunction with Personium Calendar.

Reviewer: ```suggestion
This is a sample application that recommends event information based on the schedules and profile information accumulated on PDS in conjunction with Personium Calendar.
```
